learners-guild/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+echo "Updating Echo Chat Dependencies"
+
+echo "Git: Pulling repos"
+echo "Git: Pulling rocketchat-ls-sso"
+cd ../rocketchat-lg-sso
+git pull origin master
+
+echo "Git: Pulling rocketchat-ls-slash-commands"
+cd ../rocketchat-lg-slash-commands
+git pull origin master
+
```

Comment:
I am 100% absolutely positively NOT okay with this.

What if I'm making changes in those repos? If I `git pull origin master`, it's going to potentially merge-in all sorts of conflicting stuff.